src/ClientModel/SamplingDataPostProcessor.cpp

```diff
@@ -32,6 +34,59 @@ namespace orbit_client_model {
 
 namespace {
 
+// class equivalent of orbit_client_protos::CallstackInfo for use as key of absl::flat_hash_map.
```

Comment:
You're right and I tried for quite a while. If I recall correctly the problem was that then it's not possible to use the `absl::Hash` framework because it doesn't support proto repeated fields. I guess we don't have to necessarily use `absl::Hash` for this but I wanted to, I think we have the same error proneness when changing the proto if we don't.

Reply:
In particular, using the `absl::Hash` framework as it is out of the box makes it so that the result is the same as for `CallstackInfoAsPairWithLvalueRefToFrames`, which is what we want here.

Reply:
Okay, I guess its fine then.